src/com/lftechnology/java/training/niraj/stemleaf/StemLeafPlot.java

```diff
@@ -0,0 +1,88 @@
+package com.lftechnology.java.training.niraj.stemleaf;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.alina.basic.SumOfMultiples;
+
+/**
+ * 
+ * Plots stem and leaf
+ * 
+ * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+ *
+ */
+public class StemLeafPlot {
+	private final static Logger LOGGER = Logger.getLogger(SumOfMultiples.class
+			.getName());
+
+	private StemLeafPlot() {
+
+	}
+
+	public static void main(String[] args) {
+		try (Scanner scanner = new Scanner(System.in)) {
+			int dataPointLength;
+
+			LOGGER.log(Level.INFO, "Please provide the number of data points:");
+			while (true) {
+				while (!scanner.hasNextInt()) {
+					LOGGER.warning("Please enter a number:");
+					scanner.next();
+				}
+				dataPointLength = scanner.nextInt();
+				if (dataPointLength < 0) {
+					LOGGER.warning("Please provide a positive number");
+					scanner.next();
+				} else {
+					break;
+				}
+			}
+			int[] dataPoints = getDataPoints(dataPointLength, scanner);
+			StemLeaf stemLeaf = new StemLeaf(dataPoints);
+			stemLeaf.plotStemLeaf();
+
+		} catch (InputMismatchException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)